Dacaspex/Exhibit
# #365 Graph: fixed alpha color inconsistency & wrap graph colors.

#### What have I done
Fixes #365.
Fixed alpha value of first color in graph.
Also extend the color array by duplicating it so it is always at least as large as the number of datasets.

#### Testing
* Import files
* Open graph
* Select sheet
* Observe correct alpha value.
* Select at least 8 more sheets.
* Observe colors wrapping.

#### Addresses SRD section(s)
/
